scripts/operations/gateway-test/uai-gateway-test.sh

```diff
@@ -133,6 +135,8 @@ if [[ -z ${UAI_POD} ]]; then
 fi
 
 # Get a token to talk to SLS
+#shellcheck disable=SC2155
+#shellcheck disable=SC2046
```

Comment:
For future reference, you can do this:
```
#shellcheck disable=SC2155,SC2046
```

Reply:
I prefer the one per-line style, which makes reviewing changes easier.  For example, if one were to be re-enabled, but not the other 🤷